client/components/dot-pager/style.scss

```diff
@@ -17,7 +20,6 @@
 	&.is-prev,
 	&.is-next {
 		opacity: 0;
-		position: absolute;
```

Comment:
Actually, each page has the same height is because the default value of `align-items` is `normal`. According to its [definition](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items#values), it behaves as stretch on all other  absolutely-positioned boxes.
> In absolutely-positioned layouts, the keyword behaves like start on replaced absolutely-positioned boxes, and as stretch on all other absolutely-positioned boxes.

So we can set `align-items: flex-start` to avoid stretch each items.

Reply:
You're right that we don't need the `position: absolute` in this patch, but I think that's because we're grabbing the height of the current page and applying it to the container directly.

The `position: absolute` works by taking all the other cards out of the layout flow, so the `auto` height is calculated only on the current card.

The `align-items: flex-start` doesn't appear to have the same effect. When I open up the live link and remove the explicit height ( `document.getElementsByClassName( 'dot-pager__pages' )[0].style.height = ''`) the card expands to show the whitespace at the bottom again, even though `align-items: flex-start` is applied to the element.

That's fine, though - the absolute position is redundant with the explicit height, so it's fine to drop it.

I'm not sure if the `align-items: flex-start` is doing anything else, so if not we could drop it too.

Reply:
https://user-images.githubusercontent.com/13596067/127949649-a3ece6c3-5f08-486e-8975-ecbd85fe9910.mov

As you can see in the video, if we don't have `align-items: flex-start`, the height of each item is equal and stretch to max height of the card (I just use the first two for demo). If we apply the `align-items: flex-start` to the container, the height of each item doesn't stretch and equals to its content.